benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -1092,6 +1092,9 @@ void GraphicsBenchmarkApp::Render()
     }
     else
 #endif
+        // Wait for and reset render complete fence
```

Comment:
Reminder to self: this isn't going to work, consider:

- is this needed for XR? If so, can be moved up before the `
#if defined(PPX_BUILD_XR)` around line 1084?
- If this is not needed for XR then move it inside the `else` block at line 1098

Reply:
I spoke with @apazylbe and we believe that this wait is needed in both XR and non-XR cases. So I've moved it farther up to around line 1084